tests/transformer/google_test.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+def test_transformer_google():
```

Comment:
Could you add a docstring to your tests as well as breaking test cases?

Reply:
Yes, I can add a docstring. How would you model a breaking test? The only thing that I have in mind is, that the transformer would fail if we provide an invalid input file. Does this make sense to you?

Reply:
Yeah that makes total sense to me. I would add
- Files with a broken structure (test case: the retriever created a malformed file)
- Files cannot be downloaded (test case: S3 bucket is down / cannot be reached)
I want to make sure that we can catch those failures should they occur.

Reply:
True. Strike the download! We'll cover empty inputs with the rest of the tests in the follow up task!